packages/components/src/tooltip/test/index.tsx

```diff
@@ -436,4 +436,50 @@ describe( 'Tooltip', () => {
 			await waitExpectTooltipToHide();
 		} );
 	} );
+
+	describe( 'nested', () => {
+		it( 'should render the outer tooltip and ignore inner tooltips', async () => {
```

Comment:
Nit: I think we should use "nested" instead of "inner" for better clarity. 

```suggestion
		it( 'should render the outer tooltip and ignore nested tooltips', async () => {
```

Reply:
[`edb03e7` (#57202)](https://github.com/WordPress/gutenberg/pull/57202/commits/edb03e77cf32aa494870bd8bfb61c2d8adc41e8e)